### ISSUE TITLE
Sodium Support for 1.18.1 Baritone

### ISSUE THREAD

**Issue by RGBSoul** [cabaletta/baritone#3262]
## Suggestion
Adding support for Fabric Sodium for 1.18.1 versions of baritone

## Context
I think that this will make it a lot easier to run on lower-end computers
 
## Checklist
- [ ] Added Sodium Support for Baritone
- [ ] It works properly (Could be an issue with the client I'm using: Feather Client)

**Comment by wagyourtail** [comment]
wdym. it already works.
![image](https://user-images.githubusercontent.com/6234704/153691163-94d1c76d-0861-4913-b1f8-db54f4408990.png)

**Comment by RGBSoul** [comment]
> wdym. it already works. ![image](https://user-images.githubusercontent.com/6234704/153691163-94d1c76d-0861-4913-b1f8-db54f4408990.png)

Looks like it's an issue with the client I'm using, sorry for my ignorance

**Comment by RGBSoul** [comment]
So basically the issue with it is that every time I try to load the game with the mod installed with the client I'm using (Feather) it just doesn't open at all. I've tried using different Baritone API's, not using an API, and different versions of the standalone, and it just won't work. I'm not sure if it's a bug with the launcher or an anti-cheat filter. Either way, I've found out a way to make it work without using that client along with Sodium, so it's not much of an issue anymore
